atd-cr3-api/.env.example

```diff
@@ -1,8 +1,10 @@
-API_ENVIRONMENT="STAGING"
-AUTH0_DOMAIN="atd-datatech.auth0.com"
-CLIENT_ID="2qbqz2sf8L9idBOwn0d5YA9efNgQbL7c"
-AWS_DEFALUT_REGION="us-east-1"
-AWS_S3_CR3_LOCATION="staging/path_to_files"
-AWS_S3_BUCKET="your_bucket_name_here"
-AWS_S3_KEY="your_s3_key_here"
-AWS_S3_SECRET="your_s3_secret_here"
\ No newline at end of file
+API_ENVIRONMENT=STAGING
+AUTH0_DOMAIN=atd-datatech.auth0.com
+CLIENT_ID=2qbqz2sf8L9idBOwn0d5YA9efNgQbL7c
+AWS_DEFALUT_REGION=us-east-1
+AWS_S3_CR3_LOCATION=staging/path_to_files
+AWS_S3_BUCKET=your_bucket_name_here
+AWS_S3_KEY=your_s3_key_here
+AWS_S3_SECRET=your_s3_secret_here
+API_CLIENT_ID=secret_here
+API_CLIENT_SECRET=secret_here
```

Comment:
NB: these two are new environment variables

Reply:
Thanks for adding these! 🙏 I happened to notice that the `API_ENVIRONMENT` is not used in the server code. We can probably get away with removing it here if you agree we should.

Reply:
Great idea; it's outta there. 🏟️ ⚾